composer.json

```diff
@@ -16,6 +16,12 @@
     "autoload-dev": {
         "classmap": [ "app/tests/WebTestCase.php" ]
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:ElectricMaxxx/sonata-admin-integration-bundle.git"
+        }
+    ],
```

Comment:
this is not needed. The branch lives in the main repository, so no need to configure composer to use your fork.